auto_remediate/config_rules.py

```diff
@@ -16,29 +39,59 @@ def rds_instance_public_access_check(self, resource_id):
         Returns:
             boolean -- True if remediation was successful
         """
-        client = boto3.client("rds")
-
-        # unfortunately the resourceId provided by AWS Config is DbiResourceId
-        # and cannot be used in the modify_db_instance function
-        # we therefore need to search all RDS instances
         try:
-            response = client.describe_db_instances()
-
+            response = self.client_rds.describe_db_instances()
+        except:
+            self.logging.error("Could not describe RDS DB Instances.")
+            return False
+        else:
             for instance in response.get("DBInstances"):
                 if resource_id == instance.get("DbiResourceId"):
-                    client.modify_db_instance(
-                        DBInstanceIdentifier=instance.get("DBInstanceIdentifier"),
-                        PubliclyAccessible=False,
-                    )
-                    break
+                    try:
+                        self.client_rds.modify_db_instance(
+                            DBInstanceIdentifier=instance.get("DBInstanceIdentifier"),
+                            PubliclyAccessible=False,
+                        )
+                        self.logging.info(
+                            f"Disabled Public Accessibility for RDS Instance '{resource_id}'."
+                        )
+                        return True
+                    except:
+                        self.logging.error(
+                            f"Could not disable Public Accessibility for RDS Instance '{resource_id}'."
+                        )
+                        self.logging.error(sys.exc_info()[1])
+                        return False
 
+    def s3_bucket_server_side_encryption_enabled(self, resource_id):
+        """Enables Service Side Encruption for an S3 Bucket
```

Comment:
typo

Reply:
also Server-Side, not Service-Side (also in the logging of this function)

Reply:
ecb5909 and 4c4302d